Core/include/Acts/Geometry/TrackingVolume.hpp

```diff
@@ -115,6 +115,8 @@ class TrackingVolume : public Volume {
   TrackingVolume(const TrackingVolume&) = delete;
   TrackingVolume& operator=(const TrackingVolume&) = delete;
 
+  TrackingVolume(TrackingVolume&&) = default;
```

Comment:
```suggestion
  TrackingVolume(TrackingVolume&&) = default;
  TrackingVolume& operator=(TrackingVolume&&) = default;
```

do we also want a move assign?

Reply:
I didn't need one, but can add it.